jenkins/ci.suse.de/pipelines/openstack-ardana-bootstrap.Jenkinsfile

```diff
@@ -87,6 +87,23 @@ pipeline {
       }
     }
 
+    stage('ansible linting') {
+      steps {
+        sh '''
+          set -x
+
+          # Make a copy of the ansible virtualenv
+          venv=$(mktemp -d /tmp/linter-venv-XXXXX)
+          rsync -PHa /opt/ansible/* ${venv}/
+
+          # Install and run ansible-lint
+          ${venv}/bin/pip install ansible-lint
+          ${venv}/bin/ansible-lint ~ardana/openstack/ardana/ansible
+          ${venv}/bin/ansible-lint ${WORKSPACE}/automation-git/scripts/jenkins/ardana/ansible
+        '''
+      }
+    }
+
     stage('bootstrap deployer') {
```

Comment:
one final point about the ansible linter target is that you need the deployer to be fully set up in order to have the `/var/lib/ardana/scratch` area. the ansible linter stage cannot run until much later, after the bootstrap deployer stage is done, later even, right before the `site.yml` playbook is launched.